### ISSUE TITLE
Update the card hover state

### ISSUE THREAD

**Issue by fionamorrison23** [civictheme/uikit#688]
### User story

As a user with low vision or a vision impairment
I want the hover state on cards to be updated
So that the hover state is visually accessible to me and is compliant with WCAG 2.2 AA

### Description

The hover state on cards relies on colour alone and an 'elevation'. This solution is not accessible. 

### Acceptance criteria

Given I am an anonymous user
When I hover on any of the card types listed below:
- - Navigation card
- - Promo card
- - Event card
- - Service card
- - Publication card - slightly different behaviour can be seen here on the icon
Then the following styling is present:
- Entire card is a link
- Elevation effect occurs: Elevation/Elevation 2
- Border displays around entire card: 
- - Width: 1px
- - Colour: border Colours/Border
- Title colour changes to: /Interaction Colours/Interaction Background
- An underline appears on the title:
- - underline width: (to come)
- - colour: /Interaction Colours/Interaction Background
- Circular background displays behind icon:
- - Size: 32px x 32px
- - Colour: /Interaction Colours/Interaction Background
- - Opacity: 10%
